packages/web/components/complex/orders-history/index.tsx

```diff
@@ -41,6 +46,24 @@ export const OrderHistory = observer(() => {
     userAddress: wallet?.address ?? "",
   });
 
+  const claimOrders = useCallback(async () => {
+    try {
+      logEvent([EventName.LimitOrder.claimOrdersStarted]);
+      await claimAllOrders();
+      logEvent([EventName.LimitOrder.claimOrdersCompleted]);
+    } catch (error) {
+      if (error instanceof Error && error.message === "Request rejected") {
+        // don't log when the user rejects in wallet
+        return;
+      }
+      const { message } = error as Error;
+      logEvent([
+        EventName.LimitOrder.claimOrdersFailed,
+        { errorMessage: message },
```

Comment:
is this going to be useful via amplitude? To me, it seems it would be preferable to use Sentry or DataDog (which I believe we're now using) instead to capture error messages. Then, we'd get stack trace info and otherwise more info useful for troubleshooting

I don't think Amplitude is the tool to use for error messages, IMO

Reply:
We can move this to Datadog if that's preferred. Do you have any examples of the best way to do that?